### ISSUE TITLE
Define a project area to explore in Autonomous Drive

### ISSUE THREAD

**Issue by meia-autonomous** [sixauto/sixauto#1]
As a sixauto Developer I need to define and propose to the team one Autonomous Drive solution I like to explore.

**Comment by carlosseco29** [comment]
I've been thinking these days and there are two solutions I would like to explore:
1.
    OBD is a standard protocol common to all recent cars. 
    OBD stands for on-board diagnosis and can provide information about car status and failures.
    We could somehow try to predict failures earlier. I'm pretty sure that is already done -> sometimes cars go into safe mode where they don't go as fast if something's wrong in order to protect the engine, for example.
    But what if we could predict that the car will go into safe mode if you don't go to the repair shop soon?
    Sometimes a car turns on an error light (example: check engine light). There are OBD<->bluetooth adapters which allow you to connect a mobile phone to the OBD system. What if we could provide info to the car owner about what caused the error light to turn on (in layman's terms). We could also suggest car repair shops and provide an option to schedule the car for maintenance.

2.
    We could teach an AI how to drive any car

**Comment by LuisAzeved** [comment]
One idea I would like to explore:

Teach an AI to detect which person is sitting and adjust the seat "environment" based on that. In the driver's seat, it could adjust the mirrors, the seat height/tilt, the steering wheel position, the default driving mode, the radio station, the seat massage settings... In the other seats it could adjust the seat height/tilt, the seat massage settings... 
We could go further and have different adjustments for the same person based on the current status of that person. Tweaking the AC settings based on the person body temperature, change the intensity of the seat massage or change the music genre/volume based on the person's heart rate. We could have different adjustments based on the combination of people inside the car: if the driver is driving with his family the car could with to comfort driving mode, if the driving is driving with his friends the car could switch to sports mode.

Note that this are just draft ideas that could surely be improved.

**Comment by NunoTMEIA** [comment]
The first idea that i had was basically the same one that Carlos Seco suggested (On-board diagnosis).

This is another idea that i would like to explore too:
We could work with the car's exterior sensors and try to get the car to recognize different traffic signs and different traffic situations. so that it provides autonomous driving.

**Comment by NunoTMEIA** [comment]
The area chosen by unanimous decision was OBD which can provide information about car status and failures.
